task2/task2.1.js

```diff
@@ -0,0 +1,171 @@
+const { v4: get_uuid} = require('uuid');

```

Comment:
it's better to exclude `isDeleted` field for all methods. Clients shouldn't know about this field